tests/test_parsers.py

```diff
@@ -127,7 +127,7 @@ def test_parse_obographs(self):
         self.assertEqual(
             # this number went up from 8099 when the curies.Converter was introduced
             # since it was able to handle CURIE prefix and URI prefix synonyms
-            8488,
```

Comment:
this needs its own PR fix

Reply:
https://github.com/mapping-commons/sssom-py/pull/468